src/api/aux.cpp

```diff
@@ -30,9 +33,9 @@ void parse_json_file(std::vector<Token> &tokens, std::vector<Order> &orders, std
             t = static_cast<boost::multiprecision::mpf_float>(tt);
```

Comment:
Does this casting work?

Reply:
Seems that it does. Didn't think this through too much, but I ended up using this instead of the C-style cast I had in mind, i.e., `t = (boost::multiprecision::mpf_float) tt;`